src/lib.rs

```diff
@@ -238,6 +243,23 @@ impl FileAdapter {
         }
     }
 
+    /// Provide a relative dir to be applied to the paths when finding files from which to read values
+    ///
+    /// ```rust
+    /// use figment::providers::Env;
+    /// use figment_file_provider_adapter::FileAdapter;
+    /// // This provider will only at the variables prefixed with MY_APP_ and attempt to resolve them in the directory "foo".
+    /// let file_adapter = FileAdapter::wrap(Env::prefixed("MY_APP_")).relative_dir("foo");
+    /// ```
+    pub fn relative_dir<P: AsRef<Path>>(self, path: P) -> Self {
```

Comment:
Hmm, regarding naming (and the documentation above): since we can take an absolute dir, I would name this function `relative_to_dir`: it's not the directory that is relative, but the paths that are relative _to_ the dir.

Reply:
done.